src/color-picker/color-picker.scss

```diff
@@ -45,36 +45,37 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb
   width: $iui-l;
   cursor: pointer;
   border-radius: $iui-color-swatch-border-radius;
-  background-color: var(--iui-color-swatch-background);
   margin-bottom: $iui-xs;
   margin-right: $iui-xs;
-
+  position: relative;
+  background-color: var(--iui-color-swatch-background);
+  @include iui-transparent-background;
   @supports (gap: $iui-s) {
     margin-bottom: 0;
     margin-right: 0;
   }
 
-  @include themed {
+  &::after {
+    content: '';
```

Comment:
What does this do?

Reply:
Checkout [this CodePen demo I made for you](https://codepen.io/JonGraft/pen/podadWp?editors=1100).  `content` can be used to place text before or after an element.  But if you want to place a shape, you need to pass an empty content, otherwise it won't display (see the class `.broken`).